CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
+cmake_minimum_required(VERSION 3.13)
+cmake_policy(SET CMP0167 NEW)
 project(darling-dmg)
 
-cmake_minimum_required(VERSION 3.13)
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
This pins the version to 17, which is open to issues later on.
In https://github.com/darlinghq/darling-dmg/pull/107, I uses compile feature to require at least c++11.
I leave it up to CMake to select an appropriate standard.